README.md

```diff
@@ -20,7 +20,7 @@ To help with testing app updates before they're released, you can:
 
 1. Sign up to be a beta tester
    - [Android](https://play.google.com/apps/testing/com.mattermost.rnbeta)
-   - [iOS](https://testflight.apple.com/join/Q7Rx7K9P)
+   - [iOS](https://testflight.apple.com/join/Q7Rx7K9P) - Open this link from your phone
```

Comment:
@thefactremains maybe something like `Open this link from your iOS device`? I wonder if by not specifying a phone it can be inferred  that it also applies for an iPad.

Reply:
> @thefactremains maybe something like `Open this link from your iOS device`? I wonder if by not specifying to use a phone it can be inferred that it also applies for an iPad.

agreed. will update